example/jsperf/ui.js

```diff
@@ -560,10 +560,11 @@
       if (_.indexOf(fastest, bench) > -1) {
         // mark fastest
         addClass(cell, text);
```

Comment:
To prevent auto-wrapping the line between the times character and the factor itself, which would make it hard to read. This happened to me with some tests run using the customized jsperf example, so it'll happen to others too, I'm sure. ;-)  Hence the non-breaking-space.